server/command.go

```diff
@@ -1031,6 +1030,15 @@ func executeInfo(p *Plugin, c *plugin.Context, header *model.CommandArgs, args .
 	return p.responsef(header, resp)
 }
 
+func executeAbout(p *Plugin, c *plugin.Context, header *model.CommandArgs, args ...string) *model.CommandResponse {
+	text, err := command.BuildInfo(model.Manifest{Name: "Jira", Version: manifest.Version}) // TODO: Replace with the manifest from file
```

Comment:
Just curious, what needs to be done to implement this todo? `//go:embed`?

Reply:
Exactly.

Well, I did that also as part of this PR:
[`b3c5d26` (#916)](https://github.com/mattermost/mattermost-plugin-jira/pull/916/commits/b3c5d269efa80caf9528e0ed924113bbb0586cc8)